geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/wancommand/ListGatewaysCommandDUnitTest.java

```diff
@@ -369,4 +369,144 @@ public void listGatewaysShouldCorrectlyUpdateSendersConnectedCountWhenReceiverSt
         .hasRowSize(1)
         .hasColumn("Sender Count").containsExactly("0");
   }
+
+  @Test
+  public void testListGatewaySenderOnly() {
+    setupClusters();
+
+    String command = CliStrings.LIST_GATEWAY + " --" + CliStrings.LIST_GATEWAY__SHOW_SENDERS_ONLY;
+    CommandResultAssert commandAssert = gfsh.executeAndAssertThat(command).statusIsSuccess();
+    commandAssert
+        .hasTableSection("gatewaySenders").hasRowSize(4)
```

Comment:
Is it possible to extract this "4" to a constant/variable so that it's more obvious why we expect 4 rows here? This comment also applies to other uses of a hard-coded 4 in following test methods.